pm_icecon/bt/params/amsr2.py

```diff
@@ -47,6 +50,7 @@
                 wintrc=82.71,
                 wslope=0.5352,
                 wxlimt=23.34,
+                # wxlimt=21.7  for CDR-derived wx values
```

Comment:
Can we remove this? If not, we probably need to add some logic to handle this case. We shouldn't have any behavior that requires un-commenting something to enable! This could easily lead to a situation in which something gets changed for one scenario but accidentally gets applied to others where it isn't appropriate.

Reply:
Maybe the best way to keep this information is to have a different set of parameters that are identified as having come from Goddard.  As distinct from the ones we're using in the CDR.

Reply:
I've created a new parameters file amsr2_goddard.py -- with the coefficients that Goddard used.